### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by archie426**

### CODE REVIEW

**Comment by archie426** [comment]
Wrong repo lol my bad

File: Rocket.Unturned/Chat/UnturnedChat.cs [hunk 1]
```diff
@@ -172,7 +171,7 @@ public static void Say(CSteamID CSteamID, string message, Color color)
             Say(CSteamID, message, color, false);
         }
 
-         public static List<string> wrapMessage(string text)
+         public static List<string> WrapMessage(string text)
```
Comment (Trojaner) [inline Rocket.Unturned/Chat/UnturnedChat.cs:174]: may break plugins

**Comment by Trojaner** [comment]
Actually a lot of stuff may break plugins just fyi

File: Rocket.Unturned/Chat/UnturnedChat.cs (hunk 1 above)
Comment (archie426) [reply Rocket.Unturned/Chat/UnturnedChat.cs:174]: True

**Comment by archie426** [comment]
Addressed issues with obsolete attribute

**Pull request closed without merging**